File: app/src/main/java/com/robotnec/budget/core/mvp/view/TransactionsView.java
package com.robotnec.budget.core.mvp.view;


import com.robotnec.budget.core.service.aggregation.impl.TransactionAggregation;

/**
 * @author zak <zak@swingpulse.com>
 */
public interface TransactionsView extends View {
    void displayTransactions(TransactionAggregation aggregation);
}

File: app/src/main/java/com/robotnec/budget/core/mvp/view/TransactionsView.kt
package com.robotnec.budget.core.mvp.view


import com.robotnec.budget.core.service.aggregation.impl.TransactionAggregation

/**
 * @author zak zak@swingpulse.com>
 */
interface TransactionsView : View {
    fun displayTransactions(aggregation: TransactionAggregation)
}
